### DISCUSSION TITLE
The brand new programming language!

### DISCUSSION THREAD

**Discussion by grendizerh** [community/community#46090] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hello! I am here to inform you all that a new programming language has come to town!

Are you tired of writing long code in C? No worries! Try C- right now! Just head on right to this site: [https://github.com/Rayan25062011/C-](url) 

Hope you enjoy it 😉 
Have a good day!

**Comment by Faiqlatheef** [comment]
That's great
